fix: Add Jira ignored fields to Jira integration doc (#5794)
getsentry/sentry-docs: src/docs/product/integrations/issue-tracking/jira/index.mdx

Alert notifications in Sentry can be routed to many supported integrations, but by default are aimed at email. You will need to configure a project’s [**Alert Rules**](/product/alerts-notifications/alerts/) to properly route notifications to a specific integration.

### Ignored Fields

> Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields”.

Reviewer: ```suggestion
Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields” and enter a comma separated list of the Jira field IDs. 
```

> Enter a comma separated list of the Jira field IDs, which is different from the label on the form. If you need help finding the field's ID, see the Atlassian docs [here](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html).

Reviewer: ```suggestion
Field IDs are different from the labels on the issue creation form. If you need help finding a field's ID, check out the [Atlassian docs](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html).
```

Alert notifications in Sentry can be routed to many supported integrations, but by default are aimed at email. You will need to configure a project’s [**Alert Rules**](/product/alerts-notifications/alerts/) to properly route notifications to a specific integration.

### Ignored Fields

> Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Organization Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields”.

Reviewer: ```suggestion
Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields”.
```

> Type the id or name of the Jira field, not the label, into the text input box. The following section will state how to find the field id or name.

Reviewer: ```suggestion
Type the ID/name of the Jira field, not the label, into the text input box. The following section explains how to find the field ID/name.
```

> #### Find the id of the Field

Reviewer 2: I'd suggest linking to an existing Atlassian doc about this like [this](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html) (that was the first one I found, I didn't dig too deeply so there might be a better one) instead of having this section. Then the "The following section will state how to find the field id or name." sentence can be something like "If you need help finding the field's ID, see the Atlassian docs here".

Author: @ceorourke thank you for your feedback! I really appreciate it!

In regards to your first point - I mostly used the [link](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html) you mentioned, but I added step 4 (Click the “Select an element” button on the top left of the Inspect module) as I found that I couldn't get the correct ID just from clicking on it (see image below). This may just be an issue on my end though. If you are able to, could you try it and let me know if it worked for you?
<img width="1728" alt="6B83215D-2740-4B3E-8945-EE18D13A0D6E" src="https://user-images.githubusercontent.com/116035587/202266121-0e37a382-8196-469d-ac1a-a130bcac8080.png">

In regards to your second point, yes they are the same thing. I've seen it being called both a ["name"](https://getsentry.atlassian.net/browse/WOR-2280) and ["id"](https://confluence.atlassian.com/jirakb/how-to-find-id-for-custom-field-s-744522503.html). In a discussion with the docs team, I got the feedback to call it "id or name" so that it covers both cases, but if you think one makes more sense I can change it as I can see the confusion.

Reviewer 2: <img width="161" alt="Screen Shot 2022-11-16 at 12 44 02 PM" src="https://user-images.githubusercontent.com/29959063/202290094-8a62097b-61bf-4a1c-8664-09b7301747b9.png">

I tried this out and I think it has to do with very specific clicking 😅 If you get the green div it will have the customfield but if you get the blue one it doesn't. The added steps 4 and 5 were confusing until I realized this.

Reviewer 2: Yeah I think generally linking to a 3rd party if possible is better than having to maintain our own docs in case things change. 

Regarding the id or name thing, what's confusing to me is that "name" and "label" can be easily conflated, so we're telling them to not use the label but use the name ... as a user I wouldn't know what the difference is, so personally I'd go with ID but I wouldn't block this from going through on this point.

Author: Thank you for your response! I made the changes you suggested.

> #### Find the id of the Field

Reviewer: ```suggestion
#### Find the ID of the Field
```

> 1. In Jira, access a ticket that has the field that you want the id or name of
> 2. Right click the screen and select “Inspect” or click F12
> 3. Click the “Elements” tab of the Inspect module
> 4. Click the “Select an element” button on the top left of the Inspect module
>    ![Select an element button](select-element.png)
> 5. Click the field that you want the id or name of
> 6. View the name or id of the field
>    a. The name or id of the field is the last part of the ‘data-testid’ field
>       ![Jira field id or name](jira-field-id-name.png)

Reviewer: Just FYI, you can just number these all as 1. and when it builds, it will increment the numbers correctly.

```suggestion
1. In Jira, access a ticket that has the field that you want the ID/name of.
2. Right click the screen and select “Inspect” or click F12.
3. Click the “Elements” tab of the "Inspect" module.
4. Click the “Select an element” button on the top left of the "Inspect" module.
   ![Select an element button](select-element.png)
5. Click the field that you want the ID/name of.
6. Find the last part of the ‘data-testid’ field, which is the ID/name of the field.
      ![Jira field id or name](jira-field-id-name.png)
```
